src/Hintman/Suggest/TrailingNewline.hs

```diff
@@ -0,0 +1,19 @@
+module Hintman.Suggest.TrailingNewline
+       ( suggestNoTrailingNewline
+       ) where
+
+import Hintman (LineNumber)
+
+import qualified Data.Text as T
+
+
+suggestNoTrailingNewline :: (FilePath, Text) -> Maybe (FilePath, LineNumber, Text)
+suggestNoTrailingNewline (path, body) = (generateResult . reverse . dropWhile isNewline . reverse) splitBody
```

Comment:
I'm personally in favor of using `$` instead of combination of `.` in `()`

Reply:
Interesting, is there any reason or just style?

Reply:
One of the reasons: if you add more arguments to the function and now need to apply the last argument, you will have some problems with that...